tutorial/single_pendulum/README.md

```diff
@@ -0,0 +1,39 @@
+# Run single pendulum tutorial in Gazebo Garden
+
+## Run model in Gazebo Garden with YARP integration
+
+- 1st terminal:
+
+  ~~~bash
+  yarp server
+  ~~~
+
+- 2nd terminal:
+
+  ~~~bash
+  cd tutorial/single_pendulum
+  gz sim model.sdf
+  ~~~
+
+  The Gazebo GUI will open and the single pendulum should be already spawned in the scene.
+
+  ![Gazebo Sim GUI](media/gz_gui.png)
```

Comment:
Can we just upload this to an issue and reference them? So we avoid to waist precious repo space on images.

Reply:
Done here: https://github.com/robotology/gz-sim-yarp-plugins/issues/84

Media removed in commit efa8b1f43a1fe12fc8e01243ae2907ce9363cd8c